tests/RestClientTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
I don't know if mocking really works here. If maxmd doesn't work, neither does this library.

Reply:
It just tests that the abstract class works. It's testing all 3 get post and delete methods work in the base class as expected. The other pr in the proofing package tests making the delete call to maxmd.